src/styles/components/_component-card.scss

```diff
@@ -6,14 +6,14 @@
 
   .component-card {
     margin: 0;
-    border: 2px solid var(--foreground-color);
+    border: 2px solid var(--background-color);
     display: flex;
     flex-direction: column-reverse;
     height: 21.875rem; // 350px/16
     width: 100%;
     cursor: pointer;
     transition: all 150ms ease-in-out;
-    color: var(--font-color-dark);
+
```

Comment:
You can probably delete the extra line here (not that it's really hurting anything)

Reply:
done! :)